test/unity/rest_data/ethernetPort/index.json

```diff
@@ -0,0 +1,44 @@
+{
+  "indices": [
+    {
+      "url": "/api/types/ethernetPort?compact=True&fields=attributes.description,attributes.displayValue,attributes.initialValue,attributes.name,attributes.type,description,documentation,name,type",
+      "response": "type.json"
+    },
+    {
+      "url": "/api/instances/ethernetPort/spa_eth3?compact=True&fields=bond,cascadeNames,connectorType,health,id,instanceId,isLinkUp,isRDMACapable,isRSSCapable,linuxDeviceName,macAddress,mtu,name,needsReplacement,operationalStatus,parent,parentIOModule,parentStorageProcessor,portNumber,requestedMtu,requestedSpeed,sfpSupportedProtocols,sfpSupportedSpeeds,shortName,speed,storageProcessor,supportedMtus,supportedSpeeds",
+      "response": "spa_eth3.json"
+    },
+    {
+      "url": "/api/instances/ethernetPort/spb_eth3?compact=True&fields=bond,cascadeNames,connectorType,health,id,instanceId,isLinkUp,isRDMACapable,isRSSCapable,linuxDeviceName,macAddress,mtu,name,needsReplacement,operationalStatus,parent,parentIOModule,parentStorageProcessor,portNumber,requestedMtu,requestedSpeed,sfpSupportedProtocols,sfpSupportedSpeeds,shortName,speed,storageProcessor,supportedMtus,supportedSpeeds",
+      "response": "spb_eth3.json"
+    },
+    {
+      "url": "/api/instances/ethernetPort/spa_eth3/action/modify?compact=True",
+      "body": {
+        "mtuSize": 9000
+      },
+      "response": "success.json"
+    },
+    {
+      "url": "/api/instances/ethernetPort/spb_eth3/action/modify?compact=True",
+      "body": {
+        "mtuSize": 9000
+      },
+      "response": "success.json"
+    },
+    {
+      "url": "/api/instances/ethernetPort/spb_eth3/action/modify?compact=True",
+      "body": {
+        "speed": 100
+      },
+      "response": "success.json"
+    },
+    {
+      "url": "/api/instances/ethernetPort/spa_eth3/action/modify?compact=True",
+      "body": {
+        "speed": 100
+      },
+      "response": "success.json"
+    }
+  ]
+}
```

Comment:
Please help to make sure each file has an extra end of line at the end of the file.